### DISCUSSION TITLE
Slack link doesn't work?

### DISCUSSION THREAD

**Discussion by ghost** [elementary/triage#233] [Q&A] [answered]
Hi, was looking for IRC or similar for elementary so I could communicate more easily about what features I wanted to work on or get assigned for stuff. Specifically interested in elementary/code but able to work on anything. However the slack link doesn't work for some reason? Is it restricted to people who pay?

**Comment by leolost2605** [comment, marked as answer]
See elementary/website#3220. That link should still work :)

**Reply by ghost** [reply]
Thanks! That one worked
